lib/hammer_cli_foreman_remote_execution/job_invocation.rb

```diff
@@ -39,6 +39,7 @@ class InfoCommand < HammerCLIForeman::InfoCommand
       extend WithoutNameOption
       include BaseOutput
       option '--show-inputs', :flag, _('Show the complete input of the job')
+      option '--show-host-status', :flag, _('Show the status of job result for the Hosts.')
```

Comment:
The wording feels a bit off and capital H is probably not necessary

Reply:
Okay, so what to put there? Best I can do is `Show the host status.`

Reply:
Let's use what we use in the api description, so `Show job status for the hosts`